src/libs/antares/study/parts/short-term-storage/series.cpp

```diff
@@ -127,15 +127,21 @@ bool Series::saveToFolder(const std::string& folder) const
 
     bool ret = true;
 
-    auto checkWrite = [&ret, &folder](const std::string& name, const std::vector<double>& content) {
-        ret = writeVectorToFile(folder + SEP + name, content) && ret;
+    auto checkWrite = [&ret, &folder]
+        (const std::string& name, const std::vector<double>& vect, double defValue)
```

Comment:
Renaming : 
`const std::string& name` should be renamed into **filename**
`const std::vector<double>& vect` should be renamed into **timeSeries**